config.sample.yaml

```
@@ -40,28 +41,28 @@ portals:
   enableGateway: false
   # List of regexes to match a alias that can be turned into a bridge.
   aliases:
-    # This matches _purple_ followed by anything
-    "/^_purple_(.+)$/":
+    # This matches #_bifrost_ followed by anything
+    "/^_bifrost_(.+)$/":
       # Use the prpl-jabber protocol.
-      protocol: "prpl-jabber"
+      protocol: "xmpp-js"
       properties:
         # Set room to the first regex match
         room: "regex:1"
         # Set the server to be conf.localhost
-        server: "conf.localhost"
-    # For xmpp.js, please use:
+        server: "regex:2"
+      
 
 # Automatically register users with accounts if they join/get invited
 # a room with a protocol they are not bound to.
 # This is REQUIRED for xmpp.js to work.
 autoRegistration:
   enabled: true
   protocolSteps:
-# For xmpp.js, please use:
-  xmpp-js:
-    type: "implicit"
-    parameters:
-      username: "<T_LOCALPART>@matrix.localhost"
+  # For xmpp.js, please use:
+    xmpp-js:
+      type: "implicit"
+      parameters:
+        username: "<T_LOCALPART>@matrix.localhost"
```

```suggestion
        username: "<T_LOCALPART>_<T_DOMAIN>@matrix.localhost"
```
is what we have in our configs, so I assume that's what it should be